client_device/python/ReadMe.md

```diff
@@ -0,0 +1,8 @@
+```
+
+pip install -r requirements.txt
+
+```
+
```

Comment:
We need here a short notice to tell people they have to specify their ip address config.yaml.